ci/purge_expired_sessions_from_datastore.yaml

```diff
@@ -0,0 +1,44 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: google/cloud-sdk
+params:
+  SERVICE_ACCOUNT_JSON: ((gcp.service_account_json))
+  PROJECT_ID:
+  DATAFLOW_TEMPLATE_VERSION:
+  EXPIRATION_TIME_OFFSET:
```

Comment:
I think these should be defaulted to make fly executing easier as the values doesn't matter too much. Alternatively, the README should link to where you can find the dataflow template versions.